From 1e08be37d94d13175956c9d40a69957d0f136419 Mon Sep 17 00:00:00 2001
From: antznin <agodard@witekio.com>
Date: Fri, 15 May 2020 11:34:10 +0200
Subject: [PATCH] Update imx8mqevk 2

---
 include/configs/imx8mq_evk.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/configs/imx8mq_evk.h b/include/configs/imx8mq_evk.h
index f153337a89..e75c280c1a 100644
--- a/include/configs/imx8mq_evk.h
+++ b/include/configs/imx8mq_evk.h
@@ -292,6 +292,8 @@
 #define CONFIG_IMX_VIDEO_SKIP
 #endif
 
+#include "update_common.h"
+
 #if defined(CONFIG_ANDROID_SUPPORT)
 #include "imx8mq_evk_android.h"
 #endif
-- 
2.17.1

